src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.model;
+
+public class User {
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public User(String email, String password) {
+        this.email = email;
+        this.password = password;
+    }
+
+    public byte[] getSalt() {
+        return salt;
+    }
+
+    public void setSalt(byte[] salt) {
+        this.salt = salt;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }

```

Comment:
we need to define `toString()` method to log user entity in exception messages

Reply:
done